README.md

```diff
@@ -5,29 +5,48 @@ You can get started right away with [Microk8s](https://microk8s.io/) - the might
+managing ***Apache Spark*** jobs for ***Kubernetes***.
 
 ## Setup
-The spark-client snap simplifies the setup to run Spark jobs against your Kubernetes cluster. Please follow the instructions in the [setup](/docs/setup.md) section to get started.
+The spark-client snap simplifies the setup to run Spark jobs against your Kubernetes cluster. Please 
+follow the instructions in the [setup](https://discourse.charmhub.io/t/spark-client-snap-tutorial-setup-environment/8952) section to get started.
 
-Do check out the section on [config resolution](/docs/config.md) to understand how [spark-submit](/docs/submit.md) actually resolves the configuration properties coming from a diverse set of available sources.
+Do check out the section on [config resolution](https://discourse.charmhub.io/t/spark-client-snap-explanation-hierarchical-configuration-handling/8956) to understand 
+how [spark-submit](https://discourse.charmhub.io/t/spark-client-snap-tutorial-spark-submit/8953) actually resolves the configuration properties coming 
+from a diverse set of available sources.
 
 ## Play!
-Once the [setup](/docs/setup.md) is complete, please follow the Spark job [submission guide](/docs/submit.md) to validate and start utilizing your 
+Once the [setup](https://discourse.charmhub.io/t/spark-client-snap-tutorial-setup-environment/8951) is complete, 
+please create a Spark service account using the [CLI](https://discourse.charmhub.io/t/spark-client-snap-tutorial-setup-environment/8952) and follow the Spark 
+job [submission guide](https://discourse.charmhub.io/t/spark-client-snap-tutorial-spark-submit/8953) to validate and start utilizing your 
 Kubernetes cluster for big data workloads.
 
-Don't forget to check out the interactive shells for [Scala](/docs/shell.md) and [Python](/docs/pyspark.md). 
+Don't forget to check out the interactive [shells](https://discourse.charmhub.io/t/spark-client-snap-tutorial-interactive-mode/8954) for Scala and Python. 
 They can save you a lot of time and debugging effort for authoring Spark jobs in the Kubernetes environment.
 
-Check out the [How-Tos](/docs/howto.md) section for a list of useful commands that will make your life easy working with the Spark client. 
+Check out the [How-Tos](https://discourse.charmhub.io/t/spark-client-snap-how-to-manage-spark-accounts/8959) section for a list of useful 
+commands that will make your life easy working with the Spark client. If Python is your thing, then you can 
+also manage your service accounts via a Python [API](https://discourse.charmhub.io/t/spark-client-snap-how-to-python-api/8958)
 
-If you already have a _**Charmed Kubernetes**_ setup, check out the sections for using _**spark-client**_ with [Charmed Kubernetes](/docs/charmed_k8s.md) as snap and [Charmed Kubernetes From Pod](/docs/charmed_k8s_pod.md) within the pod.
+If you already have a _**Charmed Kubernetes**_ setup, check out the sections for 
```

Comment:
incongruous line break

Reply:
as before, it does not really matter and I tried to break line such that they can be fit nicely on an IDE